apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/app/observer/TrimObserverTest.java

```diff
@@ -148,4 +231,139 @@ public BlockEventType value() {
             }
         };
     }
+
+    private int simulateFireBlockPushed(int height) {
```

Comment:
Weld is already enabled, use blockEvent.fire

Reply:
I don't see any benefit from firing event or calling method directly here. Direct call works the same way.

Reply:
Code is moved into another test class.